docs/en/latest/admin-api.md

```diff
@@ -555,7 +559,7 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 
 1. when it is `vars`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html), without the prefix '$'.
 1. when it is `header`, the `key` is required. It is equal to "http_`key`".
-1. when it is `cookie`, the `key` is required. It is equal to "cookie_`key`".
+1. when it is `cookie`, the `key` is required. It is equal to "cookie_`key`". Note: The cookie name is case-sensitive. For example: "cookie\_`x_foo`" and "cookie\_`X_Foo`" means to get different `cookie`.
```

Comment:
Please remove quotes in "cookie\_`x_foo`" and the other. And this sentence looks strange to me. I doubt it has a grammar problem but I am not sure.

Reply:
Updated.